src/Tactic/RingSolver/Examples.agda

```diff
@@ -0,0 +1,65 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Some simple use cases of the ring solver
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Tactic.RingSolver.Examples where
+
+open import Algebra
+
+open import Data.Integer.Base using (ℤ)
+open import Data.Integer.Properties
+open import Data.Maybe
+
+open import Agda.Builtin.Equality using (_≡_ ; refl)
+
+open import Tactic.RingSolver
+open import Tactic.RingSolver.Core.AlmostCommutativeRing
+
+
+module IntegerExamples where
```

Comment:
You could add these examples to the existing [README.Tactic.RingSolver](https://github.com/agda/agda-stdlib/blob/master/README/Tactic/RingSolver.agda)?